uint16 window         # window size for averaging
---
bool output             # error indicator